.gitignore

```diff
@@ -86,3 +86,4 @@ venv
 env
 .venv
 .env
+_skbuild
```

Comment:
Where is the `_skbuild` coming from? This one should have been only in skbuild and not skbuild-core. Need to double-check.